src/PanelsView.tsx

```diff
@@ -98,7 +122,7 @@ export const PanelsView = () => {
           <TabPanel height="100%" overflowY="auto">
             <ActorsPanel />
           </TabPanel>
-          <TabPanel height="100%" overflowY="auto">
+          <TabPanel height="100%" overflowY="auto" hidden={embed.isEmbedded}>
```

Comment:
this could not only be hidden - this could not even be rendered (based on this condition)

Reply:
What do you mean?

Reply:
I think he's saying that we don't have to render this tab at all if embedded

Reply:
right. I just kept everything as hidden because, in some places, we send events to some actors that affect the entire logic. Though keeping everything but hiding, for now, might be a safe trade-off until we take those bits of logic out of React.

Reply:
ok fair enough. I changed all hidden elements to be conditionally excluded from the DOM.